client/src/pages/Filtering.js

```diff
@@ -17,6 +17,12 @@ const Input = styled.input`
   }
 `;
 
+const Image = styled.img`
+  width: 46%;
+  float: left;
+  margin-right: 10px;
+`;
```

Comment:
This works and therefor isn't a merge blocker.  However, I'd really like to see flexbox used here instead of percentage-based widths and floats.

Reply:
ok will update

Reply:
Updated using flexbox